[2022-06-27 07:05:00,648] {taskinstance.py:1043} INFO - Dependencies all met for <TaskInstance: first_airflow_dag.get_datetime scheduled__2022-06-27T07:04:00+00:00 [queued]>
[2022-06-27 07:05:00,659] {taskinstance.py:1043} INFO - Dependencies all met for <TaskInstance: first_airflow_dag.get_datetime scheduled__2022-06-27T07:04:00+00:00 [queued]>
[2022-06-27 07:05:00,660] {taskinstance.py:1249} INFO - 
--------------------------------------------------------------------------------
[2022-06-27 07:05:00,660] {taskinstance.py:1250} INFO - Starting attempt 1 of 1
[2022-06-27 07:05:00,660] {taskinstance.py:1251} INFO - 
--------------------------------------------------------------------------------
[2022-06-27 07:05:00,673] {taskinstance.py:1270} INFO - Executing <Task(BashOperator): get_datetime> on 2022-06-27 07:04:00+00:00
[2022-06-27 07:05:00,679] {standard_task_runner.py:52} INFO - Started process 894 to run task
[2022-06-27 07:05:00,682] {standard_task_runner.py:79} INFO - Running: ['***', 'tasks', 'run', 'first_***_dag', 'get_datetime', 'scheduled__2022-06-27T07:04:00+00:00', '--job-id', '59', '--raw', '--subdir', 'DAGS_FOLDER/second_demo.py', '--cfg-path', '/tmp/tmp6obfumra', '--error-file', '/tmp/tmpyco944ym']
[2022-06-27 07:05:00,683] {standard_task_runner.py:80} INFO - Job 59: Subtask get_datetime
[2022-06-27 07:05:00,734] {logging_mixin.py:109} INFO - Running <TaskInstance: first_***_dag.get_datetime scheduled__2022-06-27T07:04:00+00:00 [running]> on host f32f6d4577d5
[2022-06-27 07:05:00,781] {taskinstance.py:1448} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=first_***_dag
AIRFLOW_CTX_TASK_ID=get_datetime
AIRFLOW_CTX_EXECUTION_DATE=2022-06-27T07:04:00+00:00
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2022-06-27T07:04:00+00:00
[2022-06-27 07:05:00,782] {subprocess.py:62} INFO - Tmp dir root location: 
 /tmp
[2022-06-27 07:05:00,783] {subprocess.py:74} INFO - Running command: ['bash', '-c', 'date']
[2022-06-27 07:05:00,797] {subprocess.py:85} INFO - Output:
[2022-06-27 07:05:00,799] {subprocess.py:89} INFO - Mon Jun 27 07:05:00 UTC 2022
[2022-06-27 07:05:00,799] {subprocess.py:93} INFO - Command exited with return code 0
[2022-06-27 07:05:00,818] {taskinstance.py:1288} INFO - Marking task as SUCCESS. dag_id=first_***_dag, task_id=get_datetime, execution_date=20220627T070400, start_date=20220627T070500, end_date=20220627T070500
[2022-06-27 07:05:00,854] {local_task_job.py:154} INFO - Task exited with return code 0
[2022-06-27 07:05:00,878] {local_task_job.py:264} INFO - 0 downstream tasks scheduled from follow-on schedule check
